src/Scene/Scene.js

```diff
@@ -180,15 +159,49 @@ define('Scene/Scene', [
         this.gfxEngine.update();
     };
 
-    Scene.prototype.wait = function(timeWait) {
+    Scene.prototype.notifyChange = function() {
```

Comment:
* add documentation for this method
* Maybe `scene.needsUpdate()` is clearer than `scene.notifyChange()`? (or `scene.invalidate()`?)